### PR TITLE
Fix rpcbind inactive in check_upgraded_service after migration

### PR DESCRIPTION
**Pull request by hjluo**
In yast2-nfs-server we disabled the rpcbind and rpcbind.socket
service. This will cause the following rpcbind test to fail. We need
to cleanup all yast2-nfs-server related configure and restart the
rpcbind and rpcbind.socket service in check_upgraded_service.

- Related ticket:  https://progress.opensuse.org/issues/97259
- Needles: N/A
- Verification run: 

 https://openqa.nue.suse.com/tests/7234239 
 https://openqa.nue.suse.com/tests/7234273
  https://openqa.nue.suse.com/tests/7234274

### CODE REVIEW

File: lib/nfs_common.pm [hunk 1]
```diff
@@ -277,9 +277,16 @@ sub check_y2_nfs_func {
...
-        zypper_call 'rm yast2-nfs-server nfs-kernel-server', timeout => 480;
-        script_run ':> /etc/exports';
+        zypper_call 'rm yast2-nfs-server', timeout => 480;
+        # remove added nfs entry from /etc/fstab and /etc/exports
+        assert_script_run 'sed -i \'/srv/d\' /etc/fstab';
+        assert_script_run 'sed -i \'/srv/d\' /etc/exports';
         script_run 'rm -fr /srv/*';
+        script_run 'rm -fr /tmp/nfs';
+        # we need to restart rpcbind and rpcbind.socket for rpcbind test
+        systemctl('restart nfs-server',     ignore_failure => 1);
```
Comment (tinawang123) [inline lib/nfs_common.pm:287]: Add ignore_failure, if restart fail, it cannot be noticed, right?

File: lib/nfs_common.pm [hunk 2]
```diff
@@ -277,9 +277,16 @@ sub check_y2_nfs_func {
     stop_service();
     # we need to cleanup the nfs settings after service check was done.
     if ($stage eq 'after') {
-        zypper_call 'rm yast2-nfs-server nfs-kernel-server', timeout => 480;
-        script_run ':> /etc/exports';
+        zypper_call 'rm yast2-nfs-server', timeout => 480;
```
Comment (lemon-suse) [inline lib/nfs_common.pm:280]: Why need remove the yast2-nfs-server? Will this affect later test?

File: lib/nfs_common.pm [hunk 3]
```diff
@@ -277,9 +277,16 @@ sub check_y2_nfs_func {
...
+        # remove added nfs entry from /etc/fstab and /etc/exports
+        assert_script_run 'sed -i \'/srv/d\' /etc/fstab';
+        assert_script_run 'sed -i \'/srv/d\' /etc/exports';
         script_run 'rm -fr /srv/*';
+        script_run 'rm -fr /tmp/nfs';
+        # we need to restart rpcbind and rpcbind.socket for rpcbind test
+        systemctl('restart nfs-server',     ignore_failure => 1);
+        systemctl('restart rpcbind',        ignore_failure => 1);
+        systemctl('restart rpcbind.socket', ignore_failure => 1);
```
Comment (lemon-suse) [inline lib/nfs_common.pm:289]: It's better to verify the status of these services using systemctl('is-active' xxx);

File: lib/nfs_common.pm (hunk 2 above)
Comment (hjluo) [reply lib/nfs_common.pm:280]: this is the nfs configure yast2 GUI, do not need it anymore.

File: lib/nfs_common.pm (hunk 1 above)
Comment (hjluo) [reply lib/nfs_common.pm:287]: that's for the debugging. now changed.

File: lib/nfs_common.pm [hunk 4]
```diff
@@ -277,9 +277,19 @@ sub check_y2_nfs_func {
     stop_service();
     # we need to cleanup the nfs settings after service check was done.
     if ($stage eq 'after') {
-        zypper_call 'rm yast2-nfs-server nfs-kernel-server', timeout => 480;
-        script_run ':> /etc/exports';
+        zypper_call 'rm yast2-nfs-server', timeout => 480;
```
Comment (rfan1) [inline lib/nfs_common.pm:280]: Do we still need timeout =480 here?

**Review by tinawang123 — APPROVED** [review]
LGTM

File: lib/nfs_common.pm (hunk 4 above)
Comment (hjluo) [reply lib/nfs_common.pm:280]: leave it as it was. to make sure it will be 100% finished

Comment (rfan1) [reply lib/nfs_common.pm:280]: cool, thanks

**Review by rfan1 — APPROVED** [review]
LGTM

**Pull request merged**